### DISCUSSION TITLE
Real Luzern Emmen

### DISCUSSION THREAD

**Discussion by Amand1993** [mampfes/hacs_waste_collection_schedule#2592] [Q&A]
Hello, unfortunately I cannot add my calendar. Emmen Lucerne, Switzerland. The code is even in the example. but the program shows an error.
![456576139_1588221815237513_6780367132904156009_n](https://github.com/user-attachments/assets/01d0b4bb-d5a3-4677-bc3e-906648117745)

**Comment by 5ila5** [comment]
They changed their website, I updated the source in #2596 

This will be part of the next release or is available now as master (git) version 

install master/git: Developer tools -> actions -> select `update: Install update` -> click `Coose entity` and select WCS -> select `version` and type `master` in the input field -> run the action -> restart HA

install master with HACS version < 2.0: (3-dot menu -> redownload -> change version to master -> fully restart HA (this will increase update cycle by a lot)
